src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/external/SentenceCalculationType.kt

```diff
@@ -5,66 +5,65 @@ import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.AbstractSente
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.RecallType
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.RecallType.FIXED_TERM_RECALL_14
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.RecallType.FIXED_TERM_RECALL_28
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.RecallType.STANDARD_RECALL
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.RecallType.STANDARD_RECALL_255
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.SopcSentence
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.StandardDeterminateSentence
 
+private val UNSUPPORTED: Class<out AbstractSentence>? = null
+
 // These SentenceCalculationType values come from NOMIS - they map to offender_sentences.sentence_calc_type in NOMIS
 enum class SentenceCalculationType(
+  val sentenceClazz: Class<out AbstractSentence>? = StandardDeterminateSentence::class.java,
   val recallType: RecallType? = null,
-  val sentenceClazz: Class<out AbstractSentence> = StandardDeterminateSentence::class.java,
   val primaryName: String? = null,
-  val isFixedTermRecall: Boolean = false,
-  val isSupported: Boolean = true,
-  val isIndeterminate: Boolean = false,
   val isSDS40Eligible: Boolean = false,
-  val isToreraEligible: ToreraEligibilityType = ToreraEligibilityType.NONE,
+  val toreraEligibilityType: ToreraEligibilityType = ToreraEligibilityType.NONE,
   val sdsPlusEligibilityType: SDSPlusEligibilityType = SDSPlusEligibilityType.NONE,
 ) {
-  ADIMP(isSDS40Eligible = true, isToreraEligible = ToreraEligibilityType.SDS, sdsPlusEligibilityType = SDSPlusEligibilityType.SDS),
-  ADIMP_ORA(isSDS40Eligible = true, isToreraEligible = ToreraEligibilityType.SDS, sdsPlusEligibilityType = SDSPlusEligibilityType.SDS),
-  YOI(isSDS40Eligible = true, isToreraEligible = ToreraEligibilityType.SDS, sdsPlusEligibilityType = SDSPlusEligibilityType.SDS),
-  YOI_ORA(isSDS40Eligible = true, isToreraEligible = ToreraEligibilityType.SDS, sdsPlusEligibilityType = SDSPlusEligibilityType.SDS),
+  //region SDS / ORA Sentences
+  ADIMP(isSDS40Eligible = true, toreraEligibilityType = ToreraEligibilityType.SDS, sdsPlusEligibilityType = SDSPlusEligibilityType.SDS),
+  ADIMP_ORA(isSDS40Eligible = true, toreraEligibilityType = ToreraEligibilityType.SDS, sdsPlusEligibilityType = SDSPlusEligibilityType.SDS),
+  YOI(isSDS40Eligible = true, toreraEligibilityType = ToreraEligibilityType.SDS, sdsPlusEligibilityType = SDSPlusEligibilityType.SDS),
+  YOI_ORA(isSDS40Eligible = true, toreraEligibilityType = ToreraEligibilityType.SDS, sdsPlusEligibilityType = SDSPlusEligibilityType.SDS),
+  SEC250(isSDS40Eligible = true, toreraEligibilityType = ToreraEligibilityType.SDS, sdsPlusEligibilityType = SDSPlusEligibilityType.SECTION250),
+  SEC250_ORA(isSDS40Eligible = true, toreraEligibilityType = ToreraEligibilityType.SDS, sdsPlusEligibilityType = SDSPlusEligibilityType.SECTION250),
   SEC91_03(isSDS40Eligible = true),
```

Comment:
isSDS40Eligible can probably be replaced with check if the sentence clazz is StandardDeterminateSentence